examples/launch_height_benchmark_sagemaker.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think you need two files here: `launch_height_benchmark_sagemaker`, which launches experiment with SageMaker backend locally, and then `launch_height_benchmark_sagemaker_remote`, which launces experiment with SM backend remotely.

Reply:
This file launches experiments with SageMaker backend locally and examples/launch_height_benchmark_remotely.py launches experiment with SM backend remotely.